src/components/MonthCalendar.spec.tsx

```diff
@@ -0,0 +1,133 @@
+import MonthCalendar from './MonthCalendar';
+import { mountWithProviders } from '../utils/testing';
+import { ZetkinEvent } from '../types/zetkin';
+
+describe('MonthCalendar', () => {
+    let dummyMonth : number;
+    let dummyYear: number;
```

Comment:
I may be reading this wrong so please correct me if this sounds odd to you, but I don't think there is any reason to have these variables, at least not here.

The data in these variables are just plain numbers, defaults for which are set in `beforeEach()` and then potentially overridden in the `it()` tests. But you don't _have to have_ variables for things like this.

I would just use literals in the tests, e.g.

```tsx
<MonthCalendar events={ dummyEvents } month={ 4 } year={ 2021 } />,
```

That way when I'm reading a test it's more clear to me what the values are.

The `dummyEvents` one is different, because it's complex data loaded from an external source for every test, so it's nice to have it in `beforeEach()` which means that the variable need to be declared here to be available both for loading (in `beforeEach()`) and for using it (in `it()`).